Dart/src/com/jetbrains/lang/dart/ide/runner/server/vmService/DartVmServiceDebugProcess.java

```diff
@@ -147,7 +150,6 @@ public void logError(final String message) {
           return;
         }
 
-        getSession().getConsoleView().print("Error: " + message + "\n", ConsoleViewContentType.ERROR_OUTPUT);
```

Comment:
update:

- I restored the print
- changed the warn to an error
- and adjusted the filter (to ignore the warning) to better match the removeBreakpoint call, which is what was triggering this